### PR TITLE
Lma 46

### PR DESCRIPTION
**Pull request by ylatuya**

### CODE REVIEW

File: LongoMatch.Core/Common/Enums.cs [hunk 1]
```diff
@@ -528,4 +528,16 @@ public enum OperatingSystemID
...
+	public enum FileChangeType
+	{
+		/// <summary>
+		/// A new file is added.
+		/// </summary>
+		Created,
+		/// <summary>
+		/// An existing file is deleted.
+		/// </summary>
+		Deleted,
```
Comment (vguzmanp) [inline LongoMatch.Core/Common/Enums.cs]: Is it possible to detect file modifications?

Comment (ylatuya) [inline LongoMatch.Core/Common/Enums.cs]: Yes, but right now we only support create/delete/remove.

Comment (vguzmanp) [inline LongoMatch.Core/Common/Enums.cs]: Remove == rename?
So a rename is two changes, one deletion & one creation, right?

Comment (ylatuya) [inline LongoMatch.Core/Common/Enums.cs]: rename. And a rename is handled as a Deleted + Created

File: LongoMatch.Services/ImportMonitorServices.cs [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+		{
+			// Event from the monitor usually comes from non-UI threads.
+			Config.GUIToolkit.Invoke ((s, e) => {
+				if (eventType == FileChangeType.Created) {
+					string ext = Path.GetExtension (path);
+					try {
+						if (ImportTeams && ext == Constants.TEAMS_TEMPLATE_EXT) {
+							Team team = FileStorage.RetrieveFrom<Team> (path);
+							Config.TeamTemplatesProvider.Add (team);
+							File.Delete (path);
```
Comment (vguzmanp) [inline LongoMatch.Services/ImportMonitorServices.cs]: This wil then send a FileChangedEvent (Deleted, path)

Comment (ylatuya) [inline LongoMatch.Services/ImportMonitorServices.cs]: I don't think that's an issue, the message would be correctly sent if the file is deleted :)

**Pull request merged**